app/components/sbom/component-details/summary/index.ts

```diff
@@ -52,7 +48,7 @@ export default class SbomScanDetailsComponentDetailsSummaryComponent extends Com
 
 declare module '@glint/environment-ember-loose/registry' {
   export default interface Registry {
-    'Sbom::ScanDetails::ComponentDetails::Summary': typeof SbomScanDetailsComponentDetailsSummaryComponent;
-    'sbom/scan-details/component-details/summary': typeof SbomScanDetailsComponentDetailsSummaryComponent;
+    'Sbom::ComponentDetails::Summary': typeof SbomScanDetailsComponentDetailsSummaryComponent;
+    'sbom/component-details/summary': typeof SbomScanDetailsComponentDetailsSummaryComponent;
```

Comment:
Negligible also. For this and other relocated files in this PR, the component names `SbomScanDetailsComponentDetails...` should be `SbomComponentDetails...`.

Reply:
done